app/components/vote-overview.ts

```diff
@@ -9,12 +9,6 @@ interface ArgsInterface {
   vote: VoteModel;
 }
 
-interface FormattedTableVote {
```

Comment:
(nitpick, no need to change: this seems unrelated to this PR so ideally it would have been split into a separate PR.)